src/app/views/communities/view/overview/overview.ts

```diff
@@ -220,8 +220,6 @@ export default class RouteCommunitiesViewOverview extends BaseRouteComponent {
 
 		if (this.channel === 'featured' && this.communityState.unreadFeatureCount > 0) {
 			return true;
-		} else if (this.channel === 'all' && this.communityState.unreadChannels.length > 0) {
```

Comment:
lets do `if (this.channel === 'all') return false` here. I know it works, but it looks like a bug otherwise.